src/components/Navabar/Navbar.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Navbar from './Navbar.component';
+
+describe('<Navbar />', () => {
+  beforeEach(() => {
+    render(<Navbar />);
+  });
+
+  test('Render the search input', () => {
+    const search = screen.getByPlaceholderText(/Search/i);
+    expect(search).toBeInTheDocument();
+  });
+  test('Render the dark mode text', () => {
+    const darkMode = screen.getByText(/Dark Mode/i);
+    expect(darkMode).toBeInTheDocument();
+  });
+});
```

Comment:
Good test, you are validating UI that is expected to be, good work. 

Consider to add tests of user interaction like clicks, type, change, etc.

Try to do all your tests like this one :D